### DISCUSSION TITLE
Gehackt?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1994] [Q&A]
![image](https://user-images.githubusercontent.com/84818934/197563692-685879bc-e369-4f60-91ad-b0a026ec3894.png)

**Comment by Zefau** [comment]
Das ist keine jarvis Datei, oder liegt die bei dir im `www` Ordner unter `js` ? 
Entspricht jedenfalls nicht der Namenskonvention mit `-` (statt `.`) und auch der Name `main` ist ungewöhnlich, da bei jarvis immer `index` als Name gilt.

**Reply by mcuiobroker** [reply]
Kommt anscheinend vom Regenradar.
